lib/Lift/Lift.cpp

```diff
@@ -102,13 +102,28 @@ bool LiftPass::runOnModule(llvm::Module &M) {
   return false;
 }
 
+using JumpTargetMap = std::map<MetaAddress, bool>;
+
+template<>
+struct llvm::yaml::CustomMappingTraits<JumpTargetMap> {
+  static void inputOne(IO &IO, StringRef Key, JumpTargetMap &Data) {
+    MetaAddress Address = MetaAddress::fromString(Key);
+    IO.mapRequired(Key.str().c_str(), Data[Address]);
+  }
+
+  static void output(IO &IO, JumpTargetMap &Data) {
+    for (auto &[Key, Value] : Data)
+      IO.mapRequired(Key.toString().c_str(), Value);
+  }
+};
```

Comment:
We have `llvm::yaml::ScalarTraits` for `MetaAddress`, I don't think you need to handle this by hand with `CustomMappingTraits`.